src/components/structures/IndicatorScrollbar.js

```diff
@@ -131,17 +131,29 @@ export default class IndicatorScrollbar extends React.Component {
             // the harshness of the scroll behaviour. Should be a value between 0 and 1.
             const yRetention = 1.0;
 
-            // Check for trackpad users every so often to avoid boosting their scroll.
-            // See https://github.com/vector-im/riot-web/issues/10005
+            // whenever see horizontal scrolling, assume the user is on a trackpad
+            // for at least the next 15 minutes.
```

Comment:
ideally we maintain a link to the issue so we don't screw it up again

Reply:
we do, a few lines below